tests/skipCrossGenFiles.arm.txt

```diff
@@ -0,0 +1,2 @@
+mscorlib.dll
+System.Runtime.WindowsRuntime.UI.Xaml.dll
```

Comment:
@parjong I have faced no problem when running runtest.sh crossgen to generating *.ni.dll file. We have a little runtime issue to use crossgen, but it's different problem.